### ISSUE TITLE
Installation problem with pip install hope

### ISSUE THREAD

**Issue by vegabook** [Pyrlang/Pyrlang#73]
As per the [documentation](https://pyrlang.github.io/Pyrlang/build-library.html) on Python 3.8 on Ubuntu 20.04:

![image](https://user-images.githubusercontent.com/3780883/101995850-041a7a00-3cc5-11eb-99a6-788775158841.png)

![image](https://user-images.githubusercontent.com/3780883/101995861-172d4a00-3cc5-11eb-8278-65273375ab03.png)

**Comment by s2hc-johan** [comment]
Hi, yea we preped for it, but then I didn't get it up. Sorry I'll try to get it up again. you can check it out and install locally, if you want speed you want to have rust installed as well

**Comment by vegabook** [comment]
@s2hc-johan okay so I'll install rust and then python setup.py install?

I think this is such an awesome library it would be great if the docs were up to date. Happy to do it once I get it working. Especially now that erlport [seems dead](https://github.com/hdima/erlport). 

If I may say though it's really important that docs reflect project status otherwise this is is going to turn away new users bigtime. But perhaps you're not really interested in this project growing in a big way?

**Comment by kevin-olbrich** [comment]
> Hi, yea we preped for it, but then I didn't get it up. Sorry I'll try to get it up again.

Is a Pypi package still in the works? I have a bad feeling when installing modules manually, as i have several CI tasks that do not work correctly without version info. :-)

**Comment by s2hc-johan** [comment]
Yes it is, what os are you using? Term is binary or you need rust installed to get it performant

**Comment by kevin-olbrich** [comment]
I'm using Debian 10 and I plan to use Term inside Django.
